### DISCUSSION TITLE
PA01 指引中cmake指令未更新

### DISCUSSION THREAD

**Discussion by syan2018** [BoomingTech/Piccolo#346] [Announcements]
使用PA01中 cmake -S engine/ -B build 时，会跳过主目录cmakelist下的 set(PILOT_ROOT_DIR "${CMAKE_CURRENT_SOURCE_DIR}") 
导致后续编译shader时include(${PILOT_ROOT_DIR}/cmake/ShaderCompile.cmake)出错

帮同学们小踩一个坑🌿注意使用 cmake -S . -B build

**Comment by Ol6rin** [comment]
非常感谢！我们会尽快修正文档中的错误！
